### ISSUE TITLE
Yolo + SiamMask

### ISSUE THREAD

**Issue by OleksiiYeromenko** [foolwood/SiamMask#196]
Is there a way to provide Yolo bbox detections as input for SiamMask?

**Comment by outfielder** [comment]
Yes you can - I have done the same thing before using the [Ultralytics object detection API](https://github.com/ultralytics/yolov3) and this [paper ](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=9500218) explores just that. 

Bear in mind though that the SiamMask codebase is not really written with much in the way of maintainability and so is not the easiest to work with. I may open source my implementation of it later on when I have some free time.

**Comment by ikauwsp** [comment]
Hi sorry do you still have the code for integrating detection via YOLO as an input for the SiamMask model? Would love to take a look at it for reference! Thank you!
